src/test/pybind/test_rados.py

```diff
@@ -1371,7 +1371,7 @@ def test_monmap_dump(self):
 
     @pytest.mark.bench
     def test_osd_bench(self):
-        cmd = dict(prefix='bench', size=4096, count=8192)
+        cmd = dict(prefix='bench', size=4096, count=8192, object_size=2147483648, object_num=9999)
```

Comment:
This is a common change, please move this into a separate commit in order to explain the rationale.

Reply:
it shouldn't get into that PR at all, removing it (it was for debugging only)